Makefile

```diff
@@ -43,7 +42,8 @@ HOST := 10.223.0.1
 NETMASK := 16
 DEFAULT_NETWORK_FILE := 10-eth0.network
 
-.EXPORT_ALL_VARIABLES:
+# TODO : figure out why we are exporting all variables
```

Comment:
If you dont, then things like the EULA acceptance go interactive on you and your build hangs...